src/Transports.AspNetCore/WebSockets/GraphQLWs/MessageType.cs

```diff
@@ -0,0 +1,155 @@
+using GraphQL.Transport;
+
+namespace GraphQL.Server.Transports.AspNetCore.WebSockets.GraphQLWs;
+
+/// <summary>
+/// Protocol message types defined in
+/// <see href="https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md"/>
+/// and
```

Comment:
Why did you mention https://github.com/enisdenjo/graphql-ws/blob/master/src/common.ts ? It's implementation details for JS platform, not protocol bits.

Reply:
I think it had more error code IDs than were listed in the protocol.